README.md

```diff
@@ -331,6 +331,31 @@ func main() {
 }
 ~~~
 
+### [Mux](https://github.com/gorilla/mux)
+~~~ go
+//main.go
+package main
+
+import (
+    "log"
+    "net/http"
+    
+    "github.com/gorilla/mux"
+    "github.com/unrolled/secure" // or "gopkg.in/unrolled/secure.v1"
+)
+
+func main() {
+    secureMiddleware := secure.New(secure.Options{
+        FrameDeny: true,
+    })
+    
+    r := mux.NewRouter()
+    r.Use(secureMiddleware.Handler)
+    http.Handle("/", r)
+    log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), nil))
```

Comment:
We should either define `port` or just set it as a hardcoded value for the example.

Reply:
Great catch! Done! https://github.com/klische/secure/commit/f6aaf6c141fcc612065cffbed62850fe61ed01ac